serialization/publish.gradle

```diff
@@ -0,0 +1,52 @@
+apply plugin: 'com.novoda.bintray-release'
+apply plugin: 'maven'
+
+if (project.getGradle().startParameter.taskNames.any{it.contains('bintrayUpload')}) {
+    group = "com.twitter.serial"
+    version = project.VERSION_NAME
+
+    gradle.taskGraph.whenReady { taskGraph ->
+        taskGraph.getAllTasks().find{it.path == ":$project.name:generatePomFileForMavenPublication"}.doLast {
+            file("build/publications/maven/pom-default.xml").delete()
+            pom {
+                project {
+                    name "Serial"
+                    artifactId "serial"
+                    packaging project.name == 'compiler' ? 'jar' : 'aar'
+                    description 'Light-weight, fast framework for object serialization in Java, with Android support.'
+                    url 'https://github.com/twitter/serial'
+                    version project.VERSION_NAME
+
+                    scm {
+                        url 'git@github.com:twitter/Serial.git'
+                        connection 'scm:git:git@github.com:twitter/Serial.git'
+                    }
+
+                    licenses {
+                        license {
+                            name 'The Apache Software License, Version 2.0'
+                            url 'http://www.apache.org/licenses/LICENSE-2.0'
+                        }
+                    }
+
+                    developers {
+                        developer {
+                            id 'twitter'
+                            name 'Twitter'
+                            url 'https://www.twitter.com/'
+                        }
+                    }
+                }
+            }.writeTo("build/publications/maven/pom-default.xml")
+        }
+    }
+}
+
+publish {
```

Comment:
All these values can be moved to gradle.properties

Reply:
Ah okay, I wasn't sure what should go there, so I just put stuff I expected would change, i.e. just the version name